en/docs/learn/provisioning-users-to-hubspot.md

```diff
@@ -0,0 +1,154 @@
+
+    ```
+        Username: admin
+        Password: admin
+        User Endpoint: https://localhost:8253/hubspot/contacts
+    ```
+
+    ![scim-provisioning-conf](../assets/img/tutorials/scim-provisioning-conf.png)
+
+1. Click Update and save the configuration
+
+#### Step 03: Configuring the outbound provisioning
+
+Now we need to enable sending out provisioning requests to the previously configured provisioning system for any user 
+management operation done via the management console, SOAP API or the SCIM API. To do that, you must configure outbound 
+provisioning identity providers against the resident service provider. So, based on the outbound configuration, 
+users created in WSO2 Identity Server can also be provisioned to external systems like HubSpot or Google Apps.
+
+1. Log on to the [Management Console](https://localhost:9443/carbon/) using the username and password.
+2. In the **Main** menu under the **Identity** section, click **Resident** under **Service Providers.**
+3. In the resulting screen, expand the **Outbound Provisioning Configuration** section.
+4. In the **Outbound Provisioning Configuration** section, do the following.
+
+    1. Select the identity provider you added from the drop-down menu available and click the + sign to add it. If you 
+    have not added an identity provider yet, this step is not possible.
+    2. Once added, the identity provider is displayed as an entry in the list. Select scim from the drop-down to ensure 
+    that the SCIM operation is used for provisioning.
+    3. If the option, Blocking is enabled, the outbound provisioning request will be blocked until the response is 
+    received. By default, the request would be non-blocking.
+    4. If the option, Enable Rules is enabled, the outbound provisioning request will be executed along with the 
+    XACML rules enabled.
+    5. Click Update to save your configurations.
+
+    ![outbound-provisioning-conf](../assets/img/tutorials/outbound-provisioning-conf.png)
+
+## Try it out
+
+1. Access WSO2 Identity Server [User Portal](https://localhost:9443/user-portal/).
+2. Click **Create Account**.
+
+    ![sign-in](../assets/img/tutorials/sign-in.png)
+
+3. Enter the user name and click **Proceed to Self Register**.
+
+    ![start-sign-in](../assets/img/tutorials/start-sign-in.png)
+
+1. The **Create New Account** screen appears.
+
+    ![create-new-account](../assets/img/tutorials/create-new-account.png)
+
+1. Enter the **First Name** and **LastName**.
+2. Enter a preferred password in the **Password** and **Confirm password** text boxes.
+3. Enter your email address in the **Email** text box and the phone number in the Phone **Numbers** text box.
+4. You may enter other details in the given text boxes.
+5. Read the Privacy Policy and select the **Privacy Policy** checkbox.
+6. Click **Register**.
+7. Login to HubSpot.
+8. On the navigation panel, click contacts. You will see that the user you created in WSO2 IS has also been created in 
```

Comment:
**Contacts**

Reply:
Fixed https://github.com/wso2/docs-is/pull/1758/commits/d61920ada49084e1d9a38ac45aceaf204f1c7b36